evalml/tests/component_tests/test_per_column_imputer.py

```diff
@@ -190,6 +190,15 @@ def test_non_numeric_valid(non_numeric_df):
     assert_frame_equal(X_expected, X_t)
 
 
+def test_datetime_doesnt_error(fraud_100):
```

Comment:
nit: I would also maybe specify a column to impute such as `amount` or `provider`